test/unit/statistics_announcement_test.rb

```diff
@@ -99,12 +99,12 @@ class StatisticsAnnouncementTest < ActiveSupport::TestCase
     announcement.update!(publishing_state: "unpublished", redirect_url: "https://www.test.alphagov.co.uk/foo")
   end
 
-  test 'a redirect item is published to content-store after being unpublished' do
+  test 'a redirect item is published to Publishing API after being unpublished' do
     announcement = create(:statistics_announcement)
 
-    Whitehall::PublishingApi.expects(:publish_redirect).with do |instance|
-      instance.is_a?(Whitehall::PublishingApi::Redirect) &&
-        assert_valid_against_schema(instance.as_json, "redirect")
+    Whitehall::PublishingApi.expects(:publish_redirect_async).with do |base_path, redirects|
+      base_path == announcement.public_path &&
+        redirects == [{ path: announcement.public_path, destination: '/foo', type: "exact" }]
```

Comment:
@benilovj and I agreed that there should still be a assertion against the schema - `assert_valid_against_schema`, though it may require changing the stubbing.